<?xml version="1.0"?>
<launch>
    <node pkg="points_ground_filter" type="points_ground_filter_node" name="points_ground_filter_node" output="screen">
        <param name="sub_topic" value="/regr_fusion_rslidar_points"/>
        <param name="pub_ground_topic" value="/pandar_points_ground"/>
        <param name="pub_no_ground_topic" value="/pandar_points_no_ground"/>
        
        <param name="show_points_size" value="true"/>
        <param name="show_time" value="true"/>
        
        <param name="sensor_height" value="2.0"/>
        <param name="radius_divider" value="0.15"/>
        <param name="theta_divider" value="0.1"/>
        <param name="local_slope_threshold" value="10"/>
        <param name="general_slope_threshold" value="4"/>
        
        <param name="ground_filter_mode" value="false"/>
        <param name="ground_meank" value="5"/>
        <param name="ground_stdmul" value="1.0"/>
        
        <param name="no_ground_filter_mode" value="false"/>
        <param name="no_ground_meank" value="5"/>
        <param name="no_ground_stdmul" value="1.0"/>
    </node>
</launch>
